libs/designsystem/src/lib/components/segmented-control/segmented-control.component.scss

```
@@ -21,32 +21,62 @@
     }
   }
 
+  &.default-mode {
+    ion-segment {
+      --background: #{utils.get-color('white')};
+
+      display: inline-flex; // The segmented control itself is block-level, but we don't want the background to stretch
+      width: auto; // The segmented control itself is block-level, but we don't want the background to stretch
```

```suggestion
      width: fit-content;
```

You should of course verify that my suggestion works as intended 😁

---

It indeed does, much more readable!